start-service.sh

```diff
@@ -9,6 +9,7 @@ function help() {
     echo "usage:"
     echo "  run:"
     echo "    $0 db               - start PostgreSQL database"
+    echo "    $0 manage_db        - start a PostgreSQL terminal session"
```

Comment:
it would be better to have the same format that the script name. `manage-db`

Reply:
fixed